src/Microsoft.DotNet.PlatformAbstractions/Native/PlatformApis.cs

```diff
@@ -112,11 +112,29 @@ private static DistroInfo LoadDistroInfo()
                         result.VersionId = line.Substring(11).Trim('"', '\'');
                     }
                 }
-                return result;
+
+                return NormalizeDistroInfo(result);
             }
             return null;
         }
 
+        // For some distros, we don't want to use the full version from VERSION_ID. One example is
+        // Red Hat Enterprise Linux, which includes a minor version in their VERSION_ID but minor
+        // versions are backwards compatable.
+        //
+        // In this case, we'll normalized RIDs like 'rhel.7.2' and 'rhel.7.3' to a generic
+        // 'rhel.7'. This brings RHEL in line with other distros like CentOS, Ubuntu or Debian which
```

Comment:
I re-worded the comment to this:

```
// For some distros, we don't want to use the full version from VERSION_ID. One example is
// Red Hat Enterprise Linux, which includes a minor version in their VERSION_ID but minor
// versions are backwards compatable.
//
// In this case, we'll normalized RIDs like 'rhel.7.2' and 'rhel.7.3' to a generic
// 'rhel.7'. This brings RHEL in line with other distros like CentOS or Debian which
// don't put minor version numbers in their VERSION_ID fields because all minor versions
// are backwards compatible.
```

The point I was trying to make with Ubuntu is that if you go to ubuntu.org they list the 16.04 version you can download as 16.04.2 (since it has additional patches since RTM) but that ".2" is not reflected in os-release.

I agree it's confusing, so I just dropped the ubuntu part.  Debian and CentOS are much clear examples of the pattern.